src/main/kotlin/com/manimdsl/linearrepresentation/LinearRepresentation.kt

```diff
@@ -98,68 +106,84 @@ data class ArrayElemAssignObject(
     val index: Int,
     val newElemValue: ExecValue,
     val animatedStyle: AnimationProperties?,
-    val secondIndex: Int? = null
+    val secondIndex: Int? = null,
+    val hidden: Boolean,
 ) : ManimInstr {
     override fun toPython(): List<String> {
-        val animationString = if (animatedStyle?.textColor != null) ", color=${animatedStyle.textColor}" else ""
+        val animationString = if (animatedStyle?.textColor != null) ", color=${animatedStyle.textColor.toUpperCase()}" else ""
         val assignIndex2D = if (secondIndex == null) "" else ".rows[$secondIndex]"
-        return listOf("self.play($arrayIdent$assignIndex2D.array_elements[$index].replace_text(\"${newElemValue.value}\"$animationString))")
+        return if (!hidden) {
```

Comment:
You could make a second interface (e.g., `HideableManimInstr`) to separate it from `ManimInstrWithRuntime` so that this data class can implement it and use `playIfNotHidden`

Reply:
Actually I just noticed, this class has just been missed for runtime - instead of making a new interface, you can just implement `ManimInstrWithRuntime` and add `${getRuntimeString()}` right before the final bracket here